website/content/docs/updates/release-notes.mdx

```diff
@@ -8,14 +8,53 @@ description: >-
 # Vault release notes
 
 - **Version**: 1.20.x
-- **GA date**: TBD
+- **GA date**: 2025-06-25
 
 @include 'release-notes/intro.mdx'
 
+## Executive summary
+
+Vault Enterprise 1.20.0 streamlines the user experience, and improves visibility
+and transparency around billing. The latest version of Vault also introduces new
+capabilities related to cryptography, secret recovery, and provides enhanced
+ecosystem integrations for centralizing secrets.
+
+
+### Highlights
+
+- Improves cost-saving by streamlining billing transparency and contextualizing
+  price increases.
+
+- Simplifies and enhances the user experience by improving namespace navigation,
+  providing a customizable login function, and releasing a new secret recovery
+  function.
+
+- Enhances and expands secure integrations and reduces friction on plugin
+  distribution by allowing customers to import key value-compatible secrets from
+  AWS, Azure, and GCP.
+
+- Adds SCEP protocol support in Vault PKI for certificate automation and reduces
```

Comment:
@hc-meredith I broke this item into two (one about adding SCEP support and one about the verified integrations) to simplify the language. Please let me know if this is acceptable and if there are additional language tweaks you'd like to see